### PR TITLE
Create org.gnome.ghex.json

### PR DESCRIPTION
**Pull request by bil-elmoussaoui**
Tried it on my system, works fine! 
Not sure about the build-options for the other archs

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.gnome.ghex

File: org.gnome.ghex.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "rename-icon": "ghex",
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc", "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        /* Filesystem */
+        "--filesystem=host",
+        /* dconf */
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
Comment (nedrichards) [inline org.gnome.ghex.json]: I think you need a little bit more for dconf, see here: https://github.com/flathub/org.gnome.iagno/blob/3c77a388f309ce0df25f12d23463a95edd3279c5/org.gnome.iagno.json#L18

**Comment by nedrichards** [comment]
bot, build org.gnome.ghex

**Comment by bil-elmoussaoui** [comment]
@nedrichards It seems that the bot should be asked again to build ghex

**Comment by nedrichards** [comment]
bot, build org.gnome.ghex

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.ghex

**Pull request closed without merging**